frontend/prisma/schema.prisma

```diff
@@ -12,19 +12,49 @@ generator client {
 
 datasource db {
   provider = "postgresql"
-  url      = env("DATABASE_URL")
+  url      = "postgresql://postgres:postgres@localhost:5555/DATABASE"
 }
 
-model User {
-  id        Int      @id @default(autoincrement())
-  createdAt DateTime @default(now())
-  updatedAt DateTime @updatedAt
-  email     String?  @unique
-  image     String?
-  role      Role     @default(USER)
+model Club {
+  cid      String         @id @default(uuid())
+  email    String         @unique
+  password String
+  event    Event[] // Event referencing Club, one-to-many relation
+  category ClubCategory[] // ClubCategory, many-to-many relation
+  ClubInfo ClubInfo? // Defines a one-to-one relationship, optional (idk why)
```

Comment:
should be required im pretty sure

Reply:
a club can have 0 or 1 profile essentially. if we want it to be required, we may as well just include it in the table itself